mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/presenter/DebitCardPresenter.java

```diff
@@ -27,8 +27,12 @@ public void attachView(BaseView baseView) {
     @Override
     public void verifyDebitCard(String s, String s1, String s2) {
         String otp = "0000";
-        mDebitCardView.verifyDebitCardSuccess(otp);
-//        String message = "error";
-//        mDebitCardView.verifyDebitCardError(message);
+        if((s.length() < 12) || (s.length() > 19)) {
+            String message = "";
+            mDebitCardView.verifyDebitCardError(message);
```

Comment:
There is no need for this `message` variable, after reverting your changes in `verifyDebitCardError` method then you can simply do like : `mDebitCardView.verifyDebitCardError(getString(R.string.debit_card_error_message))`

Reply:
I have tried it, but it is showing "Cannot resolve 'getString(int)'". 
![ss](https://user-images.githubusercontent.com/48084596/79536997-edbd0f80-809e-11ea-82ef-b70eb944f5c5.png)

Reply:
Oh, my mistake.  Do like this: `getContext().getString(R...)`

Reply:
showing same with getContext()

Reply:
It worked, thank you
Any other change needed?

Reply:
After committing your changes don't forget to squash your commits into a single commit. Please read these guidelines too: [How to get started](https://gist.github.com/laxyapahuja/e30921a9478bdf3a9e166a023ce1882c) and [PR guidelines](https://github.com/openMF/mobile-wallet/blob/master/.github/CONTRIBUTING.md#pull-requests)